components/demo/demo-snippet.js

```diff
@@ -84,15 +98,16 @@ class DemoSnippet extends LitElement {
 		const dirAttr = this._dir === 'rtl' ? 'rtl' : 'ltr';
 		const skeleton = this._hasSkeleton ? html`<d2l-switch text="Skeleton" ?on="${this._skeletonOn}" @change="${this._handleSkeletonChange}"></d2l-switch>` : null;
 		return html`
-			<div class="d2l-demo-snippet-demo-wrapper">
+			<div class="d2l-demo-snippet-demo-wrapper ${this._fullscreen ? 'fullscreen' : ''}">
```

Comment:
Nit: usually we use the `classMap` directive to render conditional classes, but if this works ok then it seems fine.

Reply:
Yeah it seemed excessive for managing one class on a simple boolean.